helpers/lib_tasks_docker_release.py

```diff
@@ -637,6 +696,62 @@ def docker_build_prod_image(  # type: ignore
     hlitauti.run(ctx, cmd)
 
 
+# TODO(Vlad): Refactor with the `docker_tag_push_multi_build_local_image_as_dev()`.
```

Comment:
Same comment.

I would imagine that we need to update also the doc, if there is a new target?